src/components/course/CourseRunCard.jsx

```diff
@@ -218,6 +263,7 @@ const CourseRunCard = ({
             enrollmentUrl={enrollmentUrl}
             userEnrollment={userEnrollment}
             subscriptionLicense={subscriptionLicense}
+            triggerLicenseSubsidyEvent={triggerLicenseSubsidyEvent}
```

Comment:
@iloveagent57 Based on this prop we want to trigger an Optimzely custom event in [ToDataSharingConsent.jsx](https://github.com/openedx/frontend-app-learner-portal-enterprise/blob/3b4ad00421aacb5ec8473a7a318614d327fdfc26/src/components/course/enrollment/components/ToDataSharingConsent.jsx) like we are already triggering.

Any concerns, thoughts?

Reply:
I may have misunderstood your question - are you proposing to send a new Optimizely event, or that you're relying on the existing enrollment event? I don't think you need an additional event here.  We're already pushing an enrollment event (with metadata including the course run) for the user whenever this button is clicked.  You should be able to use that same event for this new experiment.

Reply:
The existing enroll event [enterprise_learner_portal_enrollment_click](https://github.com/openedx/frontend-app-learner-portal-enterprise/blob/9d66bc963a235e79aac00ebd7998b462df38b9b9/src/utils/optimizely.js#L2) is being 
triggered for two(`TO_DATASHARING_CONSENT` and `TO_ECOM_BASKET`) cases here https://github.com/openedx/frontend-app-learner-portal-enterprise/blob/4cbb5ee8c327ec2bf72e9b6c30f2c366e7052d27/src/components/course/enrollment/EnrollAction.jsx#L54-L68 but we only want to send new event in case of `TO_DATASHARING_CONSENT` because this is the only case when a user has `license subsidy`.